### DISCUSSION TITLE
처음 로그인한 유저의 닉네임 초기 설정 domain 수정 제안

### DISCUSSION THREAD

**Discussion by Kimhan-nah** [GhostPangPang/GhostPong#75] [General]
### 기존
- '닉네임 초기 설정' API의 domain : `auth`

### 제안
- '닉네임 초기 설정' API의 domain : `users`

### 이유
- 닉네임을 설정하게 되면 새로운 user를 생성해줌
- 그래서 controller, service 함수명을 `createNickname`, `createUser`에 대해서 하나를 선정하기가 굉장히 애매함
- auth가 아닌 user domain이 맞는 것 같다!

**Comment by jis-kim** [comment]
헉 쓴 지 몰랐네요! 동의합니다잉

**Comment by Kimhan-nah** [comment]
`user`로 수정 확정되었습니다 탕탕탕
